src/test/java/com/sumologic/jenkins/jenkinssumologicplugin/BaseTest.java

```diff
@@ -30,8 +31,8 @@ public void setUp() throws Exception {
         this.serverBootstrap.registerHandler("/jenkinstest/*", handler);
         start();
 
-        serverUrl = "http://" + server.getInetAddress().getCanonicalHostName() + ":"
-                + server.getLocalPort() + "/jenkinstest/123";
+        serverUrl = Secret.fromString("http://" + server.getInetAddress().getCanonicalHostName() + ":"
```

Comment:
Does this work if Jenkins node is configured to not use HTTP?

Reply:
It was for test case use only where we create a local Jenkins server for test cases.

Reply:
In production env, jenkins server are mostly HTTPS